src/commands/app/pack.js

```diff
@@ -66,15 +64,14 @@ class Pack extends BaseCommand {
     this.log('Creating configuration files...')
     await this.createDeployYamlFile(appConfig)
     await this.addCodeDownloadAnnotation(appConfig)
+    // doing this before zip so other things can be added to the zip
+    await this.config.runHook('post-pack', { appConfig, artifactsFolder: DEFAULTS.ARTIFACTS_FOLDER })
 
     // 4. zip package phase
     this.log(`Zipping package artifacts folder '${DEFAULTS.ARTIFACTS_FOLDER}' to '${flags.output}'...`)
     await fs.remove(flags.output)
     await this.zipHelper(DEFAULTS.ARTIFACTS_FOLDER, flags.output)
 
-    // ACNA-2038
-    // TODO: fire post-pack event for Events
```

Comment:
we wont need this

Reply:
based on the internal Slack thread discussion, I'll resolve this.